src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {new Excavator(), new Bulldozer(), new Truck()};
+        for (Machine machine: machines) {
+            machine.doWork();
```

Comment:
The messages within the 'doWork' methods should be informative and indicate what type of Machine is working. Ensure that the 'doWork' method in each Machine subclass provides a clear message about the machine type.

Reply:
Fixed